compiler/rustc_lint/src/lints.rs

```diff
@@ -224,9 +225,39 @@ pub(crate) struct BuiltinConstNoMangle {
     pub suggestion: Span,
 }
 
+// This would be more convenient as `from: String` and `to: String`, but then we'll ICE for formatting a `Ty`
+// when the lint is `#[allow]`ed.
```

Comment:
I read this comment several times, but I still don't really understand it. Explain some more, perhaps? Also, the comment talks about "from"/"to" but the fields are "before"/"after", which is confusing.